src/server/deamon/index.ts

```diff
@@ -157,10 +177,18 @@ export class DaemonServer extends Dispose {
 
   private connected = () => {
     statusBar.updateDevice(undefined, true);
-    this.sendRequest({
-      id: this.currentId++,
-      method: 'device.enable',
-    });
+    this.sendRequest(
+      {
+        id: this.currentId++,
+        method: 'device.enable',
+      },
+      (message: Message) => {
+        const result = ((message.result as any[]) || []).filter((d) => d !== null);
+        if (result.length === 0) {
+          statusBar.updateDevice(undefined, false);
```

Comment:
Won’t this always show `No device` while it’s loading?

Reply:
Just tested it, it does indeed clear the loading indicator pretty much instantly.
We should do this with a timeout instead

Reply:
Have no full test yet, will continue tomorrow.